src/language-server/simple-ui.langium

```diff
@@ -66,8 +66,8 @@ Topbar:
 Paragraph:
     'paragraph' text=Expression ('{' (css+=CSSProperties(',' css+=CSSProperties)*) '}')?;
 
-Header:
-    'header' text=Expression '{' 'headerlevel:' headerlevel=INT (',' css+=CSSProperties(',' css+=CSSProperties)*)? '}';
+Heading:
+    'heading' text=Expression 'headinglevel:' headinglevel=HEADINGLEVEL ('{' (css+=CSSProperties(',' css+=CSSProperties)*) '}')?;
```

Comment:
1. The keyword 'headinglevel' could be simplified to 'level'. Since we are already calling heading it is obvious that the level refers to the heading.

2. We have to decide on a uniform notation. If we follow camelCasing then the property 'headinglevel' should be 'headingLevel'

Reply:
Yup totally agree. I'll change that and also yes we should use Camel case I'll change that too.